recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
 
 build:
     number: 0
-    skip: true    # [win]
 
```

Comment:
Need to add VC features and re-render. See this [wiki]( https://github.com/conda/conda/wiki/VC-features ).